solr/solrj/src/test/org/apache/solr/client/solrj/impl/CloudHttp2SolrClientTest.java

```diff
@@ -246,6 +257,81 @@ public void testAliasHandling() throws Exception {
         2, client.query(null, paramsWithMixedCollectionAndAlias).getResults().getNumFound());
   }
 
+  @Test
+  @LogLevel("org.apache.solr.servlet.HttpSolrCall=DEBUG")
+  public void testHttpCspPerf() throws Exception {
+
+    performTest(false, "HTTPCSPTEST", 1, 10, 15, 15);
+
+  }
+  @Test
+  @LogLevel("org.apache.solr.servlet.HttpSolrCall=DEBUG")
```

Comment:
Not a great design for the tests I admit - I wanted to reuse code.

I also added a test using Zk based CSP - to compare metrics against the http one

After making the updates - I notice http based CSC also making a call to Solr to LISTALIASES (made obvious by the updated test)

The log listening can be consolidated somewhat.